### ISSUE TITLE
[Bug] Can't Launch Anything

### ISSUE THREAD

**Issue by SaltSouls** [gorilla-devs/GDLauncher#975]
**Describe the bug**
Can't launch any modded or vanilla games using either the git or non-git versions of GDLauncher.

**To Reproduce**
1. Have Manjaro KDE
2. Install GDLauncher
3. Install instance of game/modpack
4. Try running instance
5. Lose profits

**Expected behavior**
It should just work.

**Possible solutions**
Hoping someone can help with this, I'm out of ideas.

**Screenshots**
- Modded:
![Nowork.ico](https://user-images.githubusercontent.com/36361103/124247903-2fd14600-dad7-11eb-9f5e-a1794c79fe0e.png)
- Vanilla:
![icecream_spilled.bmp](https://user-images.githubusercontent.com/36361103/124248198-72931e00-dad7-11eb-8177-5a477c42558b.png)

**Logs**
- Modded: [deadlumberjack.json](https://github.com/gorilla-devs/GDLauncher/files/6753282/debug.log)
- Vanilla: [404_flavor_not_found.html](https://github.com/gorilla-devs/GDLauncher/files/6753338/latest.log)

**Operating System:**
 - OS: Manjaro KDE
 - Kernel: 5.10.42-1-MANJARO
 - DE: Plasma 5.21.5

**Additional context**
I'm too tired for this shit and its 2 in the morning.

**Comment by HiImJulien** [comment]
Hey there!
I am having the very same same issue. My solution was to changing the JRE use. Instead, try to use the JRE which GDLauncher offers you to setup. This resolved my issues.

My guess is that the JRE provided by arch are badly configured or even faulty.

**Comment by SaltSouls** [comment]
> Hey there!
> I am having the very same same issue. My solution was to changing the JRE use. Instead, try to use the JRE which GDLauncher offers you to setup. This resolved my issues.
> 
> My guess is that the JRE provided by arch are badly configured or even faulty.

Sorry I had solved my issue and forgot to reply to my own bug report stating the fix. The problem was that the hard drive I had GDLauncher storing the instances in wasn't mounted with executable functionality. This causes GDLauncher to freak out and throw a bunch of errors leading to me being very confused. It wasn't until I tried to launch Steam that I figured out what the issue was. I would suggest the devs to create a proper crash message for this, stating the problem clearly so that users that face this issue know the proper course of action to take. Just something along the lines of 
`Can't launch instance, mounted drive needs to support executable functionality`
would save a lot of time and headaches for people that maybe made the same mistake as me.
